src/commands/importExportEntities/entityDefinition.ts

```diff
@@ -95,4 +95,16 @@ export type ImportContext = Readonly<{
   spaceIdsByOldIds: IdsMap;
 }>;
 
+/**
+ * Allows extracting a subset of `ImportContext` based on keys specified.
+ * string:object maps are simplified to string:string.
+ *
+ * @template K - A subset of keys from `ImportContext` (defaults to all keys).
+ *
+ * @example type SimplifiedContext = SimplifyContext<'workflowIdsByOldIds' | 'collectionIdsByOldIds'>;
+ */
+export type SimplifyContext<K extends keyof ImportContext = keyof ImportContext> = {
```

Comment:
nitpick: I would rename this into SimplifiedContent to not use verb.. it kind of suggest me to be a function instead of a type

Reply:
again, unused type, removed :)